storage/src/adb/any/unordered/fixed.rs

```diff
@@ -59,14 +59,12 @@ impl<
         key: K,
         value: V,
     ) -> Result<Option<Location>, Error> {
-        self.log
-            .update_key_with_op(Operation::Update(key, value))
-            .await
+        self.update_key_with_op(Operation::Update(key, value)).await
     }
 
     /// Deletes `key` from the db, returning the old location of the key if any.
     pub(crate) async fn delete_return_loc(&mut self, key: K) -> Result<Option<Location>, Error> {
```

Comment:
nit: this only has 1 usage. We could remove this helper and just call `delete_key(Operation::Delete(key))` there? Same for `update_return_loc`.